core/src/test/java/eu/fasten/core/dependents/data/DependencyTest.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.fasten.core.dependents.data;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DependencyTest {
+
+    @Test
+    public void dependencyTest() {
+        var expected = new Dependency("package", "4.11");
+        var json = expected.toJSON();
+        var actual = Dependency.fromJSON(json);
+        assertEquals(expected, expected);
+    }
+
+
+    @Test
+    public void equalsTest() {
+        Assertions.assertEquals(new Dependency("package", "4.12"), new Dependency("package", "4.12", new ArrayList<>(), "compile", false, "jar", ""));
```

Comment:
This test seems to be for Java. For example, it contains jar and compile.

Reply:
Yes, right, I removed this test since Dependency Object now has only  two attributes